homeassistant/components/dsmr/sensor.py

```diff
@@ -47,6 +51,8 @@
         ),
         vol.Optional(CONF_RECONNECT_INTERVAL, default=30): int,
         vol.Optional(CONF_PRECISION, default=DEFAULT_PRECISION): vol.Coerce(int),
+        vol.Optional(CONF_FORCE_UPDATE, default=DEFAULT_FORCE_UPDATE): cv.boolean,
```

Comment:
We no longer allow changes to the YAML configuration for platform integrations.

This integration needs to be moved into its own domain first. As this integration communicates with a device, it needs to be upgraded to use a configuration flow.

For more information see: https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision
And: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

Reply:
I see that I have some reading to do. I guess that will make the PR a bit bigger. Would you propose to reject this pull request?

Out of curiosity, you may have seen that I'm quite new to home assistant development, is it just a matter of start typing when I would want to upgrade this integration or is it good practice to start a conversation somewhere?

Reply:
In the current state, it would be rejected, yes. I would propose to use this PR to bring the integration up to an upgraded state.

Reply:
Ok, so move it to it's own domain and make a configuration flow. Would I correctly conclude that the first one is a breaking change?

Reply:
Moving an integration configuration out of YAML into a config flow, is a breaking change. But before adding those sections, start building it first 😉

Reply:
Added config flow